README.md

```diff
@@ -19,13 +17,116 @@ Demonstrator tracking chain for accelerators.
 
 ✅: exists, 🟡: work started, ⚪: work not started yet
 
-In addition, the relations between datatypes and algorithms is given in the
-(approximately commutative) diagram shown on the right. Black lines indicate
-CPU algorithms, green lines indicate CUDA algorithms, and blue lines indicate
-SYCL algorithms. Solid algorithms are ready for use, dashed algorithms are in
-development, and dotted algorithms are future goals. Data types for different
-heterogeneous platforms are contracted for legibility, and identities are
-hidden.
+The relations between datatypes and algorithms is given in the (approximately
+commutative) diagram shown below. Black lines indicate CPU algorithms, green
+lines indicate CUDA algorithms, and blue lines indicate SYCL algorithms. Solid
+algorithms are ready for use, dashed algorithms are in development or future
+goals. Data types for different heterogeneous platforms are contracted for
+legibility, and identities are hidden.
+
+```mermaid
+flowchart LR
+    subgraph clusterization [<a href='https://github.com/acts-project/traccc/blob/main/core/include/traccc/clusterization/clusterization_algorithm.hpp'>Clusterization</a>]
```

Comment:
Would there be a way of referring to the branch/tag specific version of the files? :thinking:

With GitLab you can use "prefix-less" path names in MarkDown to refer to files/directories in the same branch/tag that you're looking at for the MarkDown file. Is something similar not possible with GitHub/Mermaid?

Of course the idea would be that the documentation in a given tag would stay usable in such a setup, even after things are re-designed in the main branch.

Reply:
Sadly that's not possible with Github's current Mermaid integration. :frowning: